### ISSUE TITLE
error

### ISSUE THREAD

**Issue by HuangQinJian** [Caoliangjie/pytorch-gradcam-resnet50#6]
GuidedBackpropReLUModel   has bugs.

**Comment by HuangQinJian** [comment]
![image](https://user-images.githubusercontent.com/20330704/58759394-b6fc0a80-855b-11e9-8774-cca9883dd284.png)
![image](https://user-images.githubusercontent.com/20330704/58759399-c1b69f80-855b-11e9-94ae-bd560d4956d9.png)

**Comment by Caoliangjie** [comment]
Hi HuangQinJian, it means that the `input.grad` is None. Because the baseline network is vgg19 and there are some differences with resnet50 network.  That's my fault beacause I just want to get the feature map before so I didin't test this `class` before. If the gradients is `None`, maybe there are some questions during the whole process. You can try to solve this question first. Maybe I will have time to solve that in this week....

**Comment by HuangQinJian** [comment]
> Hi HuangQinJian, it means that the `input.grad` is None. Because the baseline network is vgg19 and there are some differences with resnet50 network. That's my fault beacause I just want to get the feature map before so I didin't test this `class` before. If the gradients is `None`, maybe there are some questions during the whole process. You can try to solve this question first. Maybe I will have time to solve that in this week....

Thanks! Look forward your rely !

**Comment by tuji-sjp** [comment]
> Hi HuangQinJian, it means that the `input.grad` is None. Because the baseline network is vgg19 and there are some differences with resnet50 network. That's my fault beacause I just want to get the feature map before so I didin't test this `class` before. If the gradients is `None`, maybe there are some questions during the whole process. You can try to solve this question first. Maybe I will have time to solve that in this week....

您好，我也遇到了同样的问题，不知道如何解决，所以想请问您知道如何解决Guide gradcam的bug吗？

**Comment by wfchenwf** [comment]
> > Hi HuangQinJian, it means that the `input.grad` is None. Because the baseline network is vgg19 and there are some differences with resnet50 network. That's my fault beacause I just want to get the feature map before so I didin't test this `class` before. If the gradients is `None`, maybe there are some questions during the whole process. You can try to solve this question first. Maybe I will have time to solve that in this week....
> 
> Thanks! Look forward your reply !

do you have fixed it?

**Comment by Caoliangjie** [comment]
Sorry for being so long to fix this error, that's my fault because I think I do have Procrastination. Guidedbackprop's bug has been fixed but there are still a bug, I will fix it later.

**Comment by Caoliangjie** [comment]
fix this bug,now we can use guided backprop cam to visualize now.
